xayn-ai/src/coi/point.rs

```diff
@@ -298,8 +298,8 @@ pub(crate) mod tests {
     }
 
     impl CoiPointConstructor for PositiveCoi {
-        fn new(id: impl Into<CoiId>, point: impl Into<Embedding>, viewed: Duration) -> Self {
-            Self::new(id, point, viewed)
+        fn new(id: impl Into<CoiId>, point: impl Into<Embedding>, _viewed: Duration) -> Self {
```

Comment:
since `viewed` is nowhere used anymore, we can also remove it from the function signature in the trait definition.